### PR TITLE
Tool recommendation node bug

### PR DESCRIPTION
**Pull request by juleengraham**

### CODE REVIEW

File: client/galaxy/scripts/components/Workflow/Editor/Node.vue [hunk 1]
```diff
@@ -23,7 +23,7 @@
                 <i class="fa fa-arrow-right" />
             </b-button>
             <b-popover :target="popoverId" triggers="hover" placement="bottom" :show.sync="popoverShow">
-                <WorkflowRecommendations :node="node" @onCreate="onCreate" />
+                <WorkflowRecommendations :node="this" :get-manager="getManager" @onCreate="onCreate" />
```
Comment (guerler) [inline client/galaxy/scripts/components/Workflow/Editor/Node.vue]: Thank you for fixing this. Please parse the `getNode` function here.

File: client/galaxy/scripts/components/Workflow/Editor/Recommendations.vue [hunk 2]
```diff
@@ -103,15 +99,16 @@ export default {
             return stepNameList.join(",");
         },
         loadRecommendations() {
-            const workflowSimple = this.node.app.to_simple();
+            const manager = this.node.manager;
```
Comment (guerler) [inline client/galaxy/scripts/components/Workflow/Editor/Recommendations.vue]: Please call the `getManager` function here to access the manager.

**Review by guerler — COMMENTED** [review]
The formatting needs to be fixed, maybe by running prettier. Other than that it looks good to me, thanks for the fix.

**Comment by guerler** [comment]
Is the failing client test related?

File: client/galaxy/scripts/components/Workflow/Editor/Node.vue [hunk 3]
```diff
@@ -23,7 +23,7 @@
                 <i class="fa fa-arrow-right" />
             </b-button>
             <b-popover :target="popoverId" triggers="hover" placement="bottom" :show.sync="popoverShow">
-                <WorkflowRecommendations :node="node" @onCreate="onCreate" />
+                <WorkflowRecommendations :node="getNode()" :get-manager="getManager" @onCreate="onCreate" />
```
Comment (guerler) [inline client/galaxy/scripts/components/Workflow/Editor/Node.vue]: Use `getNode` instead although equivalent in this case.

**Comment by juleengraham** [comment]
> Is the failing client test related?

@guerler reinstalled client dependencies and retested locally. js_unit tests now passing locally so rerunning here .

**Review by guerler — APPROVED** [review]

**Pull request merged**